package.json

```diff
@@ -10,7 +10,8 @@
   ],
   "main": "index.js",
   "scripts": {
-    "start": "node index.js"
+    "start": "node index.js",
+    "clean": "rm -rf ghostpd*"
```

Comment:
Please use a script name `clean.js` in this way the command `npm run clean` will throw an error other system es. Windows.

### clean.js

```js
const fs = require('node:fs/promises')
async function main () {
// here the code to clean

}
main().catch(err => console.error(err))
```

### package.json

```json
// ...
"scripts": {
    "clean": "node clean.js",
// ...
```